core/src/com/smeanox/games/sg002/view/GameView.java

```diff
@@ -166,8 +169,9 @@ private void renderGrid(SpriteBatch spriteBatch){
 	 * @param spriteBatch
 	 */
 	public void render(SpriteBatch spriteBatch, Player activePlayer){
-		aFieldSizeX = (Consts.fieldSizeX * Consts.devScaleY * zoom);
+		aFieldSizeX = (Consts.fieldSizeX * Consts.devScaleX * zoom);
```

Comment:
The fields should still be square and not a rectangle, so we have to use devScaleY.